src/Popover/Popover.js

```diff
@@ -14,6 +15,10 @@ class Popover extends Component {
         this.state = {
             isExpanded: false
         };
+
+        //A generated shortId as fallback, incase props.popperProps.id is unset.
+        //This ID binds the popover and it's control by 'aria-controls'.
+        this.ariaControls = shortId.generate();
```

Comment:
`ariaControls` is a bit confusing - maybe `controlId` would be better?

Reply:
Since it is not the control's ID how about `popoverId` instead? or `popoverContainerId`?

Reply:
Yeah, I think `popoverId` makes sense. It's a bit difficult to distinguish between that and `popperProps.id`, but I think it's clear enough that one comes from props and the other is defined here as a fallback.